duckduckgo/macos-browser
# Bump privacy dashboard to latest version

Task/Issue URL: https://app.asana.com/0/0/1201473146283280/f
Tech Design URL:
CC:

**Description**:

Dark Theme  | Light Theme
--- | ---
<img width="681" alt="image" src="https://user-images.githubusercontent.com/635903/145085006-2a81ee93-c41f-4cd1-987c-74282a9cc988.png"> | <img width="681" alt="image" src="https://user-images.githubusercontent.com/635903/145085056-1097eabc-f5ec-4eda-b685-2ae08cfcda31.png">


**Steps to test this PR**:
1. Go to https://permission.site/
1. Click on "Popup" permission button
2. New window should open -- close it
3. Open Privacy Dashboard and see new "Pop-ups" permission with icon

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
